### PR TITLE
added a function to split the ID each 3 digits and return formatted booking id

### LINKED ISSUE #150: Format booking number according to desings
Acceptance criteria
- booking number is divided by space into groups of three digits, just like CZ format for thousands 

screen shot 2018-05-09 at 13 41 25

Related to #146

### PR DESCRIPTION
**Pull request by edgarordonez**
closes #150 

· Added a `formatBookingId` function to receive the ID and return this formatted.<br/><br/><br/><url>LiveURL: https://kiwicomsmart-faq-qhrtkrvikd.now.sh
</url>
</url>

### CODE REVIEW

File: src/SingleBookingPage/bookingItem/Header.js [hunk 1]
```diff
@@ -11,6 +11,11 @@ type Props = {|
   booking: Header_booking,
 |};
 
+const formatBookingId = (databaseId: number): string => {
```
Comment (jalbertsr) [inline src/SingleBookingPage/bookingItem/Header.js]: you can move this function to `/helpers` we'll need it in more places

Comment (edgarordonez) [reply src/SingleBookingPage/bookingItem/Header.js]: Yes, another name would be better, or it's okay so? I also thought to add another parameter to decide the "slices". What do you think?

Thanks @jalbertsr

**Comment by jalbertsr** [comment]
@MRTNKCR @mjancik the format should be `# 6 704 418` or is the screenshot correct?

Screenshot:
screenshot

**Comment by mjancik** [comment]
It should be like this: ` # 6 704 418`

**Pull request merged**